app/controllers/v0/veteran_readiness_employment_claims_controller.rb

```diff
@@ -5,7 +5,7 @@ class VeteranReadinessEmploymentClaimsController < ClaimsBaseController
     def create
       load_user
       claim = SavedClaim::VeteranReadinessEmploymentClaim.new(form: filtered_params[:form])
-      claim.add_claimant_info(current_user) if current_user&.loa3?
```

Comment:
Note for the reviewer - I've moved this loa3 check into the method itself, because I've added some additional functionality in the method that we want to call even if the user is not loa3.